desktop-app/app/components/WebView/style.module.css

```diff
@@ -64,6 +64,16 @@
   font-size: 20px;
 }
 
+.errorHelpSuggestion {
+  position: absolute;
+  top: 25%;
+  width: 100%;
+  padding: 35px;
+  text-align: justify;
+  letter-spacing: -1.3px;
```

Comment:
Why do we need this negative letter spacing?

Reply:
with negative letter spacing
![withNegativeLetterSpacing](https://user-images.githubusercontent.com/26111211/94648791-2ff3a980-02eb-11eb-8674-b9f93475a495.png)
with null letter spacing
![withNullLetterSpacing](https://user-images.githubusercontent.com/26111211/94648793-3124d680-02eb-11eb-974f-7611a2a64bac.png)

Reply:
That's because you have used `text-align: justify`. I believe you need not set text-align and the text will align to the left and that will be just fine.

Reply:
Yeah, `null` letter space version is ok.

Reply:
I have removed the text-align and the letter-spacing.Now it looks like this
![ssl_err](https://user-images.githubusercontent.com/26111211/95402579-fc76d780-0907-11eb-9245-016ffb871026.png)